src/main/scala/io/iohk/ethereum/consensus/ethash/difficulty/ConfiguredDifficultyCalculator.scala

```diff
@@ -0,0 +1,34 @@
+package io.iohk.ethereum.consensus.ethash.difficulty
+
+import io.iohk.ethereum.consensus.difficulty.DifficultyCalculator
+import io.iohk.ethereum.domain.BlockHeader
+
+class ConfiguredDifficultyCalculator(powTargetTime: Long) extends DifficultyCalculator {
```

Comment:
Not fond of the name, `EthashDiffCalc` is also configured. Maybe `TargetTimeDiffCalc` or simply `QaDiffCalc`?

Reply:
(I don't like QaDiffCalc as this will be used on our testnet and I'm not sure for how long, not just testing purposes)